modules/waybar/center.css

```diff
@@ -0,0 +1,88 @@
+window .modules-center #workspaces button {
+    background-color: @base01;
+    color: @base05;
+}
+.modules-center #wireplumber,
+.modules-center #pulseaudio,
+.modules-center #sndio {
+    background-color: @base09;
+    color: @base05;
```

Comment:
This will still have the issue mentioned in #75 - `base05` and `base09` do not necessarily contrast.

`base00` for the `color` should work here.